validation-test/stdlib/Collection/Inputs/Template.swift.gyb

```diff
@@ -54,19 +54,15 @@ def test_methods(test):
 }%
 
 % for test in all_tests():
-// Start of file: ${test.name}
+// BEGIN ${test.name}
```

Comment:
Oh, this file was compatible with `utils/split-generated-tests`.  Unfortunate duplication with `utils/split_file.py`, I agree, but `utils/split-generated-tests` was filtering out `###sourceLocation` markers.